### PR TITLE
Adding examples for binary operations

### PR DESCRIPTION
**Pull request by abhinavsri360**
Fixes #
📝 I have added more content for the page 'Operators in Binary'. Documentation for each and every binary operator is increased and improved as well.
Changes present in the most recent commit i.e., f95df72
Sample preview:
https://deploy-preview-298--tender-ardinghelli-ebb79f.netlify.app/docs/binary2
Ref #

#### Changes done:
- [x] Task 1

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

File: docs/binary2.md [hunk 1]
```diff
@@ -17,42 +17,83 @@ nav_order: 2
...
 
 ## Addition
-
+Binary addition is similar to Decimal addition. As this addition is binary implies we cannot have a number 
+greater than 1 i.e., when you do '1+1' it gives 0 with carry 1 i.e, 10.
 ```yaml 
-1. 0 + 0 = 0
-2. 0 + 1 = 1
-3. 1 + 0 = 1
-4. 1 + 1 = 1
```
Comment (YashKumarVerma) [inline docs/binary2.md:25]: I think the operation depicting simple usage must not be removed

Comment (abhinavsri360) [reply docs/binary2.md:25]: I have already removed that, those are the removed lines. Please look closely to what I have changed 😅

**Comment by abhinavsri360** [comment]
Simple operations included. 
Deployment link: https://deploy-preview-298--tender-ardinghelli-ebb79f.netlify.app/docs/binary2
@YashKumarVerma

**Comment by abhinavsri360** [comment]
Waiting for review @YashKumarVerma

**Comment by YashKumarVerma** [comment]
Good work

**Comment by abhinavsri360** [comment]
Thank you 😊

**Pull request merged**